sdk/servicebus/Azure.Messaging.ServiceBus/tests.yml

```diff
@@ -5,7 +5,7 @@ extends:
   parameters:
     MaxParallel: 4
     ServiceDirectory: servicebus
-    SDKType: client
+    Project: Azure.Messaging.ServiceBus
```

Comment:
The CI template doesn't have the Project parameter, so I'm not sure if there is a way to limit the CI to only running the tests for a specific project. 
/cc @benbp